src/commands/dockerInstaller.ts

```diff
@@ -124,14 +119,17 @@ export class MacDockerInstaller extends DockerInstallerBase {
 }
 
 export async function showDockerInstallNotification(): Promise<void> {
-    const installMessageLinux = localize('vscode-docker.commands.dockerInstaller.installDockerInfo', 'Docker is not installed. Would you like to learn more about installing Docker?');
-    const installMessageNonLinux = localize('vscode-docker.commands.dockerInstaller.installDocker', 'Docker Desktop is not installed. Would you like to install it?');
-    const learnMore = localize('vscode-docker.commands.dockerInstaller.learnMore', 'Learn more')
-    const install = localize('vscode-docker.commands.dockerInstaller.install', 'Install')
-    const osProvider: LocalOSProvider = new LocalOSProvider();
-    const dockerInstallMessage = osProvider.os === 'Linux' ? installMessageLinux : installMessageNonLinux;
-    const confirmationPrompt: vscode.MessageItem = osProvider.os === 'Linux' ? { title: learnMore } : { title: install };
-    const response = await vscode.window.showInformationMessage(dockerInstallMessage, ...[confirmationPrompt]);
+    const isLinux = os.platform() !== 'win32' && os.platform() !== 'darwin';
```

Comment:
`osUtils.isLinux()` ?

Reply:
That'd be fine. Need to ensure `osUtils` uses `os` underneath; anything else could have problems with VSCode remoting.

Reply:
I just like the idea of having the logic of "are we on Linux/Unix?" or "are we on Windows ver X or newer" to be in one place. That way it is easy to fix it as necessary.

Reply:
I cleaned it up here and elsewhere.